docs/docs/glossary.md

```diff
@@ -186,6 +186,10 @@ Once a site has been [built](#build) by Gatsby and loaded in a web browser, [cli
 
 As part of its data layer and [build](#build) process, Gatsby will automatically **infer** a [schema](#schema), or type-based structure, based on available data sources (e.g. Markdown file nodes, WordPress posts, etc.). More control can be gained over this structure by using Gatsby's [Schema Customization API](/docs/schema-customization/).
 
+### [Infrastructure As Code](/docs/glossary/infrastructure-as-code/)
```

Comment:
```suggestion
### [Infrastructure as Code](/docs/glossary/infrastructure-as-code/)
```

I think we can refer to this as Infrastructure as Code or IaC throughout. As long as "as" isn't capitalized and we're consistent all the way through, I'm good.